lightning-invoice/src/utils.rs

```diff
@@ -218,6 +198,85 @@ where
 	}
 }
 
+/// Utility to filter channels for an invoice, and return the corresponding route hints to include
+/// in the invoice.
+///
+/// The filtering ensures that the RouteHints returned only include the highest inbound capacity
+/// channel per counterparty node for which channels are present. Channels with a lower inbound
```

Comment:
Drop "for which channels are present" since the previous part of the sentence already establishes a channel exists.